frontend/docsettings.lua

```diff
@@ -60,46 +61,54 @@ local function buildCandidates(list)
                                end
                            end)
 
-    return candidates
+    -- Paths only are used
+    local candidates_path = {}
+    for _, candidate in ipairs(candidates) do
+        table.insert(candidates_path, candidate.path)
+    end
+    return candidates_path
```

Comment:
Not sure the extra massaging is worth it, you're creating garbage (in the Lua sense) for no good reason ;).

(i.e., it might actually be a net loss to do this here).